paml/paml.ttl

```diff
@@ -355,32 +355,51 @@ paml:SampleArray rdf:type owl:Class ;
         rdfs:subClassOf paml:SampleCollection ,
             [ rdf:type owl:Restriction ;
               owl:onProperty paml:containerType ;
-              owl:allValuesFrom xsd:anyURI ; # replace with container ontology, when available
+              owl:allValuesFrom paml:ContainerSpec ;
               owl:minCardinality "1"^^xsd:nonNegativeInteger ; owl:maxCardinality "1"^^xsd:nonNegativeInteger ;
               rdfs:comment "Type of container used for storing the samples. The size and dimension may not match that of the array: it is up to execution to lay out the array in one or more containers"
             ] ,
             [ rdf:type owl:Restriction ;
               owl:onProperty paml:contents ;
-              owl:allValuesFrom xsd:string ; # need to replace this with an serializable array model
+              owl:allValuesFrom paml:PrimitiveArray ; # need to replace this with an serializable array model
```

Comment:
@jakebeal @rpgoldman This is the entry point to the declaration of the SampleArray contents as a PrimitiveArray.  I reused the OrderedPropertyValue to take care of indices.  OrderedPropertyValue can only hold Identified objects, and I chose to implement the contents in my test case with LiteralString objects of the form "A1", "A2", etc. to signify the identifier for aliquots.  We could also extend this to be a dict with attributes ala samples.json.